examples/forwarding-zone/main.tf

```diff
@@ -20,6 +20,7 @@ module "dns-forwarding-zone" {
   type       = "forwarding"
   name       = var.name
   domain     = var.domain
+  labels     = var.labels
```

Comment:
Note we increasingly prefer to just hardcode values in examples. It's clearer.

Reply:
@morgante I think this is what I have done as well
https://github.com/terraform-google-modules/terraform-google-cloud-dns/pull/15/files#diff-6a99f067edc93819d3fcbfcc3eed5b77f2f42b090aa730b0c617326ee2f20907R37-R44
```hcl
variable "labels" {
  type        = map
  description = "A set of key/value label pairs to assign to this ManagedZone"
  default = {
    owner   = "foo"
    version = "bar"
  }
}
```
Or is this something different?

Reply:
It's actually not making it a variable at all.

Ex.

```
module "dns-forwarding-zone" {
  source     = "../.."
  project_id = var.project_id
  type       = "forwarding"
  name       = var.name
  domain     = var.domain
  labels     = {
    owner = "foo"
    version = "bar"
  }
}
```